Lib/Utils/PathUtils.cs

```diff
@@ -17,15 +17,15 @@ static PathUtils()
             IsUnixFs = Path.DirectorySeparatorChar == '/';
         }
 
-        public static string Normalize(string path)
+        public static string Normalize(string path, bool removeParentFolderDirective = true)
```

Comment:
Not really needed anymore, please revert. Intent for normalize is to call it only for absolute paths.